grails-app/domain/org/pih/warehouse/requisition/Requisition.groovy

```diff
@@ -91,6 +93,8 @@ class Requisition implements Comparable<Requisition>, Serializable {
     // Intended recipient
     Person recipient
 
+    Person approvedBy
```

Comment:
Can you move it down to other related fields and add a comment that these are `approval` related fields